src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -97,40 +106,41 @@ object DepositHandler {
     def getSequenceNumber(f: File): Int =
       try {
         f.getName.split('.').last.toInt
-      } catch {
+      }
```

Comment:
Not part of this change, but actually I think catching all `Throwable`s below is also a bug. We should catch the specific error that results from having something else than an integer. (By the way we do not even check that it is positive!) Other exceptions should lead to an "internal error".